src/runtime/runtime.cs

```diff
@@ -1982,6 +1984,9 @@ internal static IntPtr PyType_GenericAlloc(IntPtr type, long n)
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern IntPtr _PyObject_GetDictPtr(IntPtr obj);
 
+        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern IntPtr _PyObject_GC_Calloc(IntPtr basicsize);
```

Comment:
It is a very bad idea to use private function. Why is it required? Can the same be done with only public functions?

Reply:
I'm afraid there's no such public function now, it's required to allocate a custom size Python GC object.